### PR TITLE
Jewish calendar integration move to config flow

### PR DESCRIPTION
**Pull request by tsvi**
## Proposed change

Add changes to Jewish calendar documentation as part of move to config flow

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#84464
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/jewish_calendar.markdown [hunk 1]
```diff
@@ -14,49 +14,34 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Jewish Calendar (`jewish_calendar`) integration displays a variety of information related to the Jewish Calendar as a variety of sensors.
+The Jewish Calendar (`jewish_calendar`) integration displays various information related to the Jewish Calendar as various sensors.
+{% include integrations/config_flow.md %}
+
 
 ## Configuration
```
Comment (c0ffeeca7) [inline source/_integrations/jewish_calendar.markdown]:
```suggestion
```
This title can be removed. The config_flow include provides a title.
image

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @tsvi :+1:

**Comment by yuvalabou** [comment]
LGTM

**Review by frenck — APPROVED** [review]
Hi there @tsvi 👋 

It seems like this PR picked up a merge conflict 😢 

Could you maybe take a look?

Thanks! 👍 

../Frenck

**Review by frenck — DISMISSED** [review]
Oopsy, wrong button. I meant to request a change of course 🤦 

need... moar... coffee... ☕️

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**